app/src/main/java/com/github/kulebin/myfoursquareapp/dataSource/UrlProcessorTypeMatcher.java

```diff
@@ -0,0 +1,36 @@
+package com.github.kulebin.myfoursquareapp.dataSource;
```

Comment:
UriProcessorTypeMatcher class removed, ProcessorType is passed as param in loadData() method;